lib/aggregate/aggregate_store.rb

```diff
@@ -53,6 +56,16 @@ def aggregated_attribute_handlers
             {}
           end
       end
+
+      private
+
+      def full_attr_handler_options(options)
+        if respond_to?(:aggregate_db_storage_type)
```

Comment:
Could we add a base method to the module so we don't have to have this conditional?